http/src/main/scala/org/http4s/blaze/http/package.scala

```diff
@@ -27,7 +35,7 @@ package object http {
       SimpleHttpResponse("OK", 200, headers, ByteBuffer.wrap(body))
 
     def Ok(body: String, headers: Headers): SimpleHttpResponse =
-      Ok(body.getBytes(UTF_8), ("Content-Type", "text/plain; charset=UTF-8")+:headers)
+      Ok(body.getBytes(UTF_8), ("content-type", "text/plain; charset=utf-8")+:headers)
```

Comment:
Heh. That indicates the quality of review you received. :-)
On Jan 14, 2015 6:00 PM, "Bryce Anderson" <notifications@github.com> wrote:

> In http/src/main/scala/org/http4s/blaze/http/package.scala
> <https://github.com/http4s/blaze/pull/12#discussion-diff-22978382>:
>
> > @@ -27,7 +35,7 @@ package object http {
> >        SimpleHttpResponse("OK", 200, headers, ByteBuffer.wrap(body))
> >
> >      def Ok(body: String, headers: Headers): SimpleHttpResponse =
> > -      Ok(body.getBytes(UTF_8), ("Content-Type", "text/plain; charset=UTF-8")+:headers)
> > +      Ok(body.getBytes(UTF_8), ("content-type", "text/plain; charset=utf-8")+:headers)
>
> http2 requires all header names to be lower case. Its malformed otherwise
> <http://http2.github.io/http2-spec/index.html#HttpHeaders>. I believe I
> make sure they are lowercase elsewhere, but why make a new string.
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/http4s/blaze/pull/12/files#r22978382>.
>